packages/jsii-python-runtime/tests/test_compliance.py

```diff
@@ -1000,6 +1002,15 @@ def test_consumer_calls_method_privateclass():
 def test_consumer_calls_method_typed_as_class():
     assert ConsumerCanRingBell().when_typed_as_class(PythonConcreteBellRinger())
 
+def test_can_pass_nested_struct_as_dict():
+    # Those shouldn't raise:
+    RootStructValidator.validate(string_prop= 'Pickle Rick!!!')
+    RootStructValidator.validate(
```

Comment:
Please add a test that validates the None case (where there is no value passed into the nested struct)

Reply:
`None` is the default value of the second argument, so this is tested in the first case here... But I can add an explicit one, too, yeah.